### DISCUSSION TITLE
Question about Facebook Page integration / Facebook Ads

### DISCUSSION THREAD

**Discussion by AwakenTheJaken** [chatwoot/chatwoot#1690] [Self hosted] [answered]
Hey guys! I had a question about the Facebook Page integration. If we are to run advertisements via Facebook and someone replies to said advertisement will this be relayed to Chatwoot? Appreciate your guys' input on this, thank you!

**Comment by pranavrajs** [comment, marked as answer]
@AwakenTheJaken We don't support post comments in Chatwoot at the moment, we support Facebook DMs only.

**Reply by AwakenTheJaken** [reply]
Thank you very much for your reply! Are there any plans to add this in the future?

**Reply by pranavrajs** [reply]
Yes, we have. We have integration Tweets, we would be adding the FB posts in similar fashion.

**Reply by AwakenTheJaken** [reply]
That is excellent news! Appreciate your time and thanks for the awesome app :)

**Reply by Haashimi** [reply]
> Yes, we have. We have integration Tweets, we would be adding the FB posts in similar fashion.
Hi @pranavrajs  did you add FB post and comments ?

**Comment by ChrisGutierrezNet** [comment]
I would love this feature as well. we manage like 3-5k comments a week.

**Comment by sojan-official** [comment]
tracking this over https://github.com/orgs/chatwoot/discussions/8455#discussioncomment-8440425
